### PR TITLE
BAP-55:Added given steps to actions in dropdown binding

### PR DESCRIPTION
**Pull request by SergeyRuzin**

### CODE REVIEW

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs [hunk 1]
```diff
@@ -65,12 +65,14 @@ public void CheckDropdownHeaders([NotNull] IDropdownWrapper wrapper, [NotNull] T
                 !behavior.Contains("not"), $"{wrapper.Caption} items are {items.Aggregate((x, y) => $"{x}, {y}")}");
         }
 
+        [Given("the \"(.*) was selected in (.*)")]
```
Comment (afomina-quantori) [inline src/Behavioral.Automation/Bindings/DropdownBinding.cs:68]: We agreed to use phrase "user" to simulate actions.
Expected binding is:
user selected \"(.*)\" in (.*)

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs [hunk 2]
```diff
@@ -65,12 +65,14 @@ public void CheckDropdownHeaders([NotNull] IDropdownWrapper wrapper, [NotNull] T
...
         [When("user selects \"(.*)\" in (.*)")]
         public void ClickOnEntry([NotNull] string entry, [NotNull] IDropdownWrapper wrapper)
         {
             wrapper.Select(entry);
         }
 
+        [Given("multiple entries were selected in (.*)")]
```
Comment (afomina-quantori) [inline src/Behavioral.Automation/Bindings/DropdownBinding.cs:75]: We agreed to use phrase "user" to simulate actions.
Expected binding is:
user selected multiple entries in (.*)

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs (hunk 1 above)
Comment (afomina-quantori) [reply src/Behavioral.Automation/Bindings/DropdownBinding.cs:68]: Please, update the Bindings.md

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs (hunk 2 above)
Comment (afomina-quantori) [reply src/Behavioral.Automation/Bindings/DropdownBinding.cs:75]: Please, update the Bindings.md

**Review by afomina-quantori — CHANGES_REQUESTED** [review]

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs (hunk 1 above)
Comment (SergeyRuzin) [reply src/Behavioral.Automation/Bindings/DropdownBinding.cs]: Fixed

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs (hunk 2 above)
Comment (SergeyRuzin) [reply src/Behavioral.Automation/Bindings/DropdownBinding.cs]: Fixed

**Review by PavelAAlexeevQ — APPROVED** [review]

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs [hunk 3]
```diff
@@ -65,12 +65,14 @@ public void CheckDropdownHeaders([NotNull] IDropdownWrapper wrapper, [NotNull] T
...
         }
 
+        [Given("user selected \"(.*) in (.*)")]
         [When("user selects \"(.*)\" in (.*)")]
         public void ClickOnEntry([NotNull] string entry, [NotNull] IDropdownWrapper wrapper)
         {
             wrapper.Select(entry);
         }
 
+        [Given("user selected multiple entries in (.*)")]
```
Comment (afomina-quantori) [inline src/Behavioral.Automation/Bindings/DropdownBinding.cs:75]: I think we need to use : symbol for consistency
`[Given("user selected multiple entries in (.*?):")]`

File: BINDINGS.md [hunk 4]
```diff
@@ -125,11 +125,13 @@ Example:
 When "Properties" autocomplete contains <string> in all elements
 
 * Select value inside dropdown:
+[Given("user selected \"(.*) in (.*)")]
 [When("user selects \"(.*)\" in (.*)")]
 Example:
 When user selects <propertyValue> in "Properties" autocomplete
 
 * Select multiple entries in dropdown with multiple choice:
+[Given("user selected multiple entries in (.*)")]
```
Comment (afomina-quantori) [inline BINDINGS.md:134]: Please, add : symbol to the end

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs [hunk 5]
```diff
@@ -65,12 +65,14 @@ public void CheckDropdownHeaders([NotNull] IDropdownWrapper wrapper, [NotNull] T
                 !behavior.Contains("not"), $"{wrapper.Caption} items are {items.Aggregate((x, y) => $"{x}, {y}")}");
         }
 
+        [Given("user selected \"(.*) in (.*)")]
```
Comment (afomina-quantori) [inline src/Behavioral.Automation/Bindings/DropdownBinding.cs:68]: Should be 
` [Given("user selected \"(.*?)\" in (.*?)")]`

File: BINDINGS.md [hunk 6]
```diff
@@ -125,11 +125,13 @@ Example:
 When "Properties" autocomplete contains <string> in all elements
 
 * Select value inside dropdown:
+[Given("user selected \"(.*) in (.*)")]
```
Comment (afomina-quantori) [inline BINDINGS.md:128]: `[Given("user selected \"(.*?)\" in (.*?)")]`

File: BINDINGS.md (hunk 4 above)
Comment (afomina-quantori) [reply BINDINGS.md:134]: `[Given("user selected multiple entries in (.*?):")]`

**Review by afomina-quantori — CHANGES_REQUESTED** [review]

File: BINDINGS.md (hunk 6 above)
Comment (SergeyRuzin) [reply BINDINGS.md]: Fixed

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs (hunk 5 above)
Comment (SergeyRuzin) [reply src/Behavioral.Automation/Bindings/DropdownBinding.cs]: Fixed

File: BINDINGS.md (hunk 4 above)
Comment (SergeyRuzin) [reply BINDINGS.md]: Fixed

File: src/Behavioral.Automation/Bindings/DropdownBinding.cs (hunk 3 above)
Comment (SergeyRuzin) [reply src/Behavioral.Automation/Bindings/DropdownBinding.cs]: Fixed

**Review by afomina-quantori — APPROVED** [review]

**Review by PavelAAlexeevQ — APPROVED** [review]

**Pull request merged**